src/man/authselect.8.adoc

```diff
@@ -363,6 +363,7 @@ The *authselect* can return these exit codes:
 * 3: Current configuration is not valid, it was edited without authselect.
 * 4: System configuration must be overwritten to activate an authselect profile, --force parameter is needed.
 * 5: Executed command must be run as root.
+* 6: No authselect and no non-authselect configuration was detected.
```

Comment:
This message looks a little confusing to me.  From the commit message, I take this to mean that "no configuration" was found.  If that's correct, it may make more sense to simply say "No configuration was detected"

Or, is there another way to word "non-authselect configuration"?    "user" or "local" configuration maybe?